pymongo/message.py

```diff
@@ -1001,33 +1003,84 @@ def unack_write(
         msg: bytes,
         max_doc_size: int,
         docs: list[Mapping[str, Any]],
+        client: MongoClient,
     ) -> Optional[Mapping[str, Any]]:
         """A proxy for Connection.unack_write that handles event publishing."""
+        command_logger = logging.getLogger("pymongo.command")
```

Comment:
Could you open a ticket to document how apps can silence and configure logging output? 

For example can they enable logging in thier own app but silence all pymongo logs? Do they need to silence each logger separately ("pymongo.command", "pymongo.pool", etc...)?

Reply:
Done: [PYTHON-4167](https://jira.mongodb.org/browse/PYTHON-4167)